src/components/Map.js

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+import GoogleMapReact from 'google-map-react';
+
+import { useCurrentLocation } from '../lib/userLocation';
+
+const API_KEY = 'AIzaSyAs_fPF3j1pcQYUU2s0WMI27zhV7oe8kks';
```

Comment:
Are we good to be tracking this API key or should it be in an .env file?

Reply:
I believe the key will need to be kept on the client unencrypted, so there’s not much benefit to keeping it out of the repo. Good question!